Performance/QPSBenchmark/Sources/QPSBenchmark/Runtime/BenchmarkServiceImpl.swift

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import GRPC
+import NIO
+
+/// Implementation of asynchronous service for benchmarking.
+final class AsyncQPSServerImpl: Grpc_Testing_BenchmarkServiceProvider {
+  /// One request followed by one response.
+  /// The server returns the client payload as-is.
+  func unaryCall(request: Grpc_Testing_SimpleRequest,
+                 context: StatusOnlyCallContext) -> EventLoopFuture<Grpc_Testing_SimpleResponse> {
+    do {
+      return context.eventLoop
+        .makeSucceededFuture(try AsyncQPSServerImpl.processSimpleRPC(request: request))
+    } catch {
+      return context.eventLoop.makeFailedFuture(error)
+    }
+  }
+
+  /// Repeated sequence of one request followed by one response.
+  /// Should be called streaming ping-pong
+  /// The server returns the client payload as-is on each response
+  func streamingCall(context: StreamingResponseCallContext<Grpc_Testing_SimpleResponse>)
+    -> EventLoopFuture<(StreamEvent<Grpc_Testing_SimpleRequest>) -> Void> {
```

Comment:
This is probably the work of swift format, but preferred style in gRPC is:

```swift
func streamingCall(
  context: StreamingResponseCallContext<Grpc_Testing_SimpleResponse>
) -> EventLoopFuture<(StreamEvent<Grpc_Testing_SimpleRequest>) -> Void> {
```

i.e. one argument per line and closing `)` on a new line (since it breaks visually splits signature a body a little better.

I'm fine with these as-is given they're not implemented so just an FYI.